doc/user/download_backup.md

```diff
@@ -67,7 +67,8 @@ If backup storage type is "PV", users need to get backup directly from PV.
 
 TODO: document how to find and mount the disk.
 
-Given the cluster name, backups are stored under dir `/var/etcd-backup/v1/${CLUSTER_NAME}/` .
+Given the cluster name, it will store (and restore) in PVC named `${CLUSTER_NAME}-pvc`,
```

Comment:
Backups will be stored to (and restore from) the PVC named as `...` under the directory `...`